components/Navigation/command-line.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+const CommandLineIcon = () => {
+  // eslint-disable-next-line max-len
+  return (
```

Comment:
No need for return statement
```suggestion
  // eslint-disable-next-line max-len
const CommandLineIcon = () => (
```

Reply:
@karelhala This is how we've always done it and it's what I see in online examples. Can you point me to a better way?